index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>subrat&ridwan</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+  <div id="container">
+    <div id="header">
+       <h1 id="title">freeCodeCamp Survey Form Survey Form</h1>
+       <p id="description"><em>Thank you for taking the time to help us improve the platform</em></p>
+    </div>
```

Comment:
Please use header tag instead of this div tag. I believe it would be better 
```suggestion
    <header id="header">
       <h1 id="title">freeCodeCamp Survey Form Survey Form</h1>
       <p id="description"><em>Thank you for taking the time to help us improve the platform</em></p>
    </header>
```